### PR TITLE
Webpack build

### PR DESCRIPTION
**Pull request by data-ux**
To run dev environment with webpack (example Paikkatietoikkuna app):

1) Run `npm install` in repo root
2) Put following into Paikkatietoikkuna Jetty ext.properties to run backend in prod mode:
```
development=false
oskari.client.version=dist/devapp
```
3) Start Jetty 
4) Run `npm run paikkis:dev` to start webpack dev server with proxy to jetty
5) Open localhost:8081 in browser

### CODE REVIEW

**Comment by ZakarFin** [comment]
Awesome :+1: Got some minor issue with drawtools:

Loader: Couldn't start bundle with id drawtools. Error was: TypeError: Cannot read property 'plugin' of null
at e.getLocalization (AbstractMapModule.js?d346:1529)
at e.setMapModule (AbstractMapModulePlugin.js?a480:76)
at e.registerPlugin (AbstractMapModule.js?d346:1380)
at e.start (instance.js?b810:86)

I'm not sure if it's related to these changes. I'll have to test some more.

**Comment by ZakarFin** [comment]
For some reason the localization file is unavailable with the dev server and running a production build doesn't include it either (the oskari_lang_[language].fi files are missing). This is why drawtools fails to start.

File: bundles/framework/divmanazer/extension/DefaultExtension.js [hunk 1]
```diff
@@ -246,7 +246,7 @@ Oskari.clazz.define('Oskari.userinterface.extension.DefaultExtension',
          * Module protocol method
          */
         getName: function () {
-            return this.getConfiguration().name;
+            return this.getConfiguration().name || 'findbycoordinates'; // FIXME – this is insane
```
Comment (ZakarFin) [inline bundles/framework/divmanazer/extension/DefaultExtension.js]: This needs to be solved inside the findbycoordinates bundle.

**Comment by data-ux** [comment]
Above issues fixed (except proxy problems).

**Pull request merged**